### PR TITLE
LPS-178680 Create Apply Filter button action

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
ci:test:sf

**Review by Elvinho — APPROVED** [review]
Good Job 👍🏾

**Review by samuelleolopes — APPROVED** [review]
Good Job! :fire:

**Review by alvarolealdl — APPROVED** [review]
LGTM

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Templates/index.tsx [hunk 1]
```diff
@@ -13,16 +13,27 @@
...
+const templateFilterInitialContext = {
+	entries: [
+		{
+			label: 'Status',
+			name: 'dueStatus',
+			value: BuildStatuses.ACTIVATED,
+		},
+	],
+	filter: {dueStatus: BuildStatuses.ACTIVATED},
+};
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Templates/index.tsx:36]: Break a line

**Review by welitonpe — APPROVED** [review]
LGTM :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Templates/index.tsx [hunk 2]
```diff
@@ -13,16 +13,28 @@
...
 import useSearchBuilder from '../../../../../hooks/useSearchBuilder';
 import i18n from '../../../../../i18n';
 import dayjs from '../../../../../util/date';
-import {BuildStatuses} from '../../../../../util/statuses';
 import useBuildTemplateActions from './useBuildTemplateActions';
 
+const templateFilterInitialContext = {
+	entries: [
+		{
+			label: 'Status',
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Templates/index.tsx:30]: Translate

```suggestion
			label: i18n.translate('status'),
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by rafaelfs98** [comment]
@kevenleone

**Comment by kevenleone** [comment]
ci:forward

**Comment by joelrocha68** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:stop

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**